client/components/common/StatEditor.tsx

```diff
@@ -188,6 +190,55 @@ const StatEditor: React.FC<Props> = ({ customSet, className }) => {
     },
   );
 
+  const quickEditStat = (baseKey: StatKey, amount: number) => {
+    dispatch({
+      type: 'edit',
+      stat: baseKey,
+      value: amount,
+    });
+    debouncedCheckAndMutate();
+  };
+
+  const getQuickStatButtons = (baseKey: BaseStatKey) => {
+    const getButtonValues = (level: number) => {
+      if (level > 150) {
+        return [200, 300];
+      } else if (level > 100) {
+        return [100, 200];
+      } else if (level > 50) {
+        return [100];
+      } else {
+        return [];
+      }
+    };
+
+    const maxStats = getMaxStat(baseKey, remainingPoints);
+    const buttonValues = getButtonValues(customSet?.level ?? 200);
+
+    return (
+      <>
+        <div css={{ display: 'flex', gap: 8 }}>
+          {buttonValues.map((value) => {
+            return (
+              <Button
+                key={`stat-button-${value}`}
+                onClick={() => quickEditStat(baseKey, value)}
+              >
+                {value}
+              </Button>
+            );
+          })}
+          <Button
+            disabled={maxStats <= 0}
```

Comment:
I could see a use for this if maxStats === 0, right? If you've put 300 int and 395 vit, then it makes sense that those are 0 for those stats.

Additionally, what if maxStats % 100 === 0? Wouldn't it be strange to have two of the same button? I'm thinking we could just get rid of it in that case?